ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -298,6 +298,13 @@ objects:
             --junit-dir /tmp/artifacts/junit -f -
         }
 
+        function run-csi-tests() {
+          oc apply -f /tmp/cluster/storageclass.yaml
+          sleep 3m
+          TEST_CSI_DRIVER_FILES="${TEST_CSI_DRIVER_FILES}" openshift-tests run "${TEST_SUITE}" \
```

Comment:
I guess we can remove all this now and just let the TEST_COMMAND in the job do its thing?

Reply:
ok I thought we still need the storage class, I'll remove it.
I'll keep just the openshift test stuff

Reply:
Are we removing something here or not?

Reply:
Forgot to resolve this comment, it is about parts that have been removed (apply of manifests)